ReactAndroid/src/main/java/com/facebook/react/views/image/ReactImageView.java

```diff
@@ -82,13 +88,26 @@ float getRadius(Bitmap source) {
             0.0f,
             mScaleType);
         sMatrix.invert(sInverse);
-        return sInverse.mapRadius(mBorderRadius);
+
+        float[] radii = cornerRadii();
+
+        return new float[]{
```

Comment:
Please refer to my comment in `cornerRadii` method. I think the same applies here